src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $container-width;
+  gap: $box-gap;
+  padding: $padding;
+  margin: auto;
+
```

Comment:
[STYLES] - Don't be greedy, add some top paddings, so the top row don't go over top of the page on hover.

when day 1 is hovered, it goes up and partly hides, maybe try bigger top padding or using flex centering+height: 100vh ?

Reply:
didn't notice, I will fix this)